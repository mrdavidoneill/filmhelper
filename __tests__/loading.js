import "@testing-library/jest-dom";
import React from "react";
import Loading from "@/components/loading";
import { render, screen } from "@testing-library/react";

describe("Loading", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<Loading />);
    expect(asFragment()).toMatchSnapshot();
  });

  it("renders a circular progress indicator", () => {
    render(<Loading />);
    expect(screen.getByRole("progressbar")).toBeInTheDocument();
  });
});
