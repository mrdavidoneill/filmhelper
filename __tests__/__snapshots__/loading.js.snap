// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders correctly 1`] = `
<DocumentFragment>
  <div
    class="flex h-screen flex-grow justify-center items-center"
    style="background-color: rgb(255, 255, 255);"
  >
    <span
      class="MuiCircularProgress-root MuiCircularProgress-indeterminate MuiCircularProgress-colorPrimary css-18lrjg1-MuiCircularProgress-root"
      role="progressbar"
      style="width: 4rem; height: 4rem;"
    >
      <svg
        class="MuiCircularProgress-svg css-1idz92c-MuiCircularProgress-svg"
        viewBox="22 22 44 44"
      >
        <circle
          class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate css-176wh8e-MuiCircularProgress-circle"
          cx="44"
          cy="44"
          fill="none"
          r="20.2"
          stroke-width="3.6"
        />
      </svg>
    </span>
  </div>
</DocumentFragment>
`;
